poetry.lock

```diff
@@ -732,21 +887,21 @@ files = [
 
 [[package]]
 name = "importlib-metadata"
-version = "6.0.0"
+version = "4.13.0"
```

Comment:
`poetry` can conflict with our dependencies, this is the reason why it's not recommended to install in global python interpreter, but instead use a venv specially for poetry.

This is what you can see here, poetry conflicts with our dependencies, although it might not have impact until someone would want to lock `importlib-metadata` to `>4.13.0` (without `=`).

---

You should not disable venv for poetry in installer, and then, just do two cache for poetry and dependencies.

P.S. I see that [poetry installed in its own venv](https://github.com/ItsDrike/setup-poetry/blob/08340adbd2cd9e2cb126bda33a8f447b1740976a/action.yml#L119), and [dependencies in its own one](https://github.com/ItsDrike/setup-poetry/blob/08340adbd2cd9e2cb126bda33a8f447b1740976a/action.yml#L225). Have you something missed up?

Reply:
Read the note in PR's description, the reason for `poetry` being a dependency here is because `tox-gh-actions` won't work without it, as it won't be able to run the `poetry` command without it being in that same environment. See: https://github.com/py-mine/mcstatus/actions/runs/4195863740/jobs/7276016803 CI (this is run from a now overwritten commit by a force push).

The ItsDrike/setup-poetry action does install poetry globally, and it works as it should. If we weren't using tox to run the tests, the poetry dependency wouldn't be required.

Reply:
It looks like tox issue, and it shouldn't be fixed like that. I also found that `tox-poetry` was deprecated, because poetry added its own support for tox. Although, when I tried to switch to that official support, it just gave more pain.

The real solution would be to switch to automatic releases, so no one from devs will use tox, and we can remove it.

Reply:
I would resolve it by opening #488.